eng/pipelines/templates/steps/template-test-generate-jobs.yml

```diff
@@ -67,5 +67,6 @@ steps:
         AzdContainerImage=$(AzdContainerImage),
         TemplateBranchName=${{ parameters.TemplateBranchName }},
         AzureLocation=${{ parameters.AzureLocation }},
-        CleanupHoursDelay=${{ parameters.CleanupHoursDelay }}
+        CleanupHoursDelay=${{ parameters.CleanupHoursDelay }},
+        USE_APIM=true
```

Comment:
I'm guessing that this is only to test the USE_APIM is working?
And to avoid running all the `USE_APIM=false` on every commit?
And once there is a full run where all gates pass, we will remove this and make the generator to duplicate all gates (by APIM)
??

Reply:
This is a temporary change to enable full runs. We'll mark a random template with APIM to have an APIM run once this passes. Duplicating all templates is too costly.